### DISCUSSION TITLE
Installation script fails when installing on Ubuntu 22.04.3

### DISCUSSION THREAD

**Discussion by Pencat89** [Security-Onion-Solutions/securityonion#11094] [2.4]
I tried running sudo soup afterwards and it cleared the logs. Thought I'd mention it

**Comment by dougburks** [comment]
Please provide more information as shown at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> What are the hardware specs of each of those nodes?
How are each of those nodes configured? (ex. manager with 2 search nodes and 3 forward nodes)
Does so-status show all services running?
Do you get any failures when you run sudo salt-call state.highstate?
Does the SOC Grid page show any failures?
Provide applicable logs. If you are having problems right after setup, provide /root/sosetup.log. If you are having problems during soup, provide /root/soup.log. If you are having problems with a specific component, provide that component's logs from /opt/so/log/.
